cmd/fabric-ca-server/main_test.go

```diff
@@ -471,3 +473,55 @@ func getTestClient(port int, homeDir string) *lib.Client {
 		HomeDir: homeDir,
 	}
 }
+
+func TestConfigInit(t *testing.T) {
+	cases := []struct {
+		enabled string
+		cert    string
+		key     string
+		err     bool
+	}{
+		{
+			enabled: "true",
+			cert:    "../../testdata/tls_server-cert.pem",
+			key:     "../../testdata/tls_server-key.pem",
+			err:     true,
+		},
+		{
+			enabled: "true",
+			cert:    "noexit.pem",
+			key:     "../../testdata/tls_server-key.pem",
+			err:     false,
+		},
+		{
+			enabled: "true",
+			cert:    "../../testdata/tls_server-cert.pem",
+			key:     "noexit.pem",
+			err:     false,
+		},
+		{
+			enabled: "false",
+			cert:    "../../testdata/tls_server-cert.pem",
+			key:     "../../testdata/tls_server-key.pem",
+			err:     true,
+		},
+	}
+
+	for _, tt := range cases {
+		os.Setenv("FABRIC_CA_SERVER_OPERATIONS_TLS_ENABLED", tt.enabled)
+		os.Setenv("FABRIC_CA_SERVER_OPERATIONS_TLS_CERT_FILE", tt.cert)
+		os.Setenv("FABRIC_CA_SERVER_OPERATIONS_TLS_KEY_FILE", tt.key)
+		os.Setenv("FABRIC_CA_SERVER_BOOT", "user:pass")
```

Comment:
I'd prefer we avoid the env vars and just set appropriate values on the viper instance. There's a reason why these functions accept an instance of viper...